Makefile

```diff
@@ -110,7 +110,7 @@ nuke: clean devnet-clean
 	git clean -Xdf
 .PHONY: nuke
 
-pre-devnet: submodules
+pre-devnet: submodules update-op-geth
```

Comment:
this will modify the go.mod file to overwrite the ethereum dep to our version on op-geth